### PR TITLE
Add better SGD with decay hyper-params for MLP training

### LINKED ISSUE #13: Better SGD hyper-parameters
We should give SGD with decay better hyper-parameters so that this method is somewhat competitive in the “Putting it all into practice” section. Otherwise, it might look like this method can be ignored.

I have actually played around a bit and I think a learning rate of 0.5 and a decay 0.01 works quite well.

I was also thinking of adding the Momentum with decay example because it is just too easy to leave as an exercise and it shows that many of these techniques can be combined. Hyper-parameter values for that case that provide decent performance are a learning rate of 0.5, momentum of 0.9, and decay of 0.01.

### PR DESCRIPTION
**Pull request by ElanVB**
Resolves #13

### CODE REVIEW

**Comment by JamesAllingham** [comment]
LGTM!
